### DISCUSSION TITLE
Anyone know how to config strapi-hook-redis

### DISCUSSION THREAD

**Discussion by daddyauden** [strapi/strapi#7660] [Legacy Topics]
I install strapi-hook-redis, but don't know how to use it.

**Comment by derrickmehaffy** [comment]
Currently this package has not been updated and is not actively maintained. Most of the packages it uses are depreciated or are no longer supported by their maintainers.

See the following discussion: https://github.com/strapi/strapi/discussions/3866

We don't currently have plans to refresh this package right now, though possibly in the future.

**Reply by daddyauden** [reply]
Thanks
